packages/rspeedy/core/src/cli/build.ts

```diff
@@ -13,6 +16,55 @@ import { isCI } from '../utils/is-ci.js'
 
 export type BuildOptions = CommonOptions & {
   environment?: string[] | undefined
+  watch?: boolean | undefined
+}
+
+async function watchFiles(
```

Comment:
It looks like we are copying from `dev.ts`. Can we instead re-use the shared code?